shell/platform/android/io/flutter/plugin/platform/PlatformPlugin.java

```diff
@@ -240,21 +242,24 @@ public void onSystemUiVisibilityChange(int visibility) {
   private void setSystemChromeEnabledSystemUIMode(PlatformChannel.SystemUiMode systemUiMode) {
     int enabledOverlays;
 
-    if (systemUiMode == PlatformChannel.SystemUiMode.LEAN_BACK
-        && Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
+    Window window = activity.getWindow();
+    View view = window.getDecorView();
+    WindowInsetsControllerCompat windowInsetsControllerCompat =
+        new WindowInsetsControllerCompat(window, view);
+
+    if (systemUiMode == PlatformChannel.SystemUiMode.LEAN_BACK) {
```

Comment:
Why don't we check for the build version here anymore?

Reply:
A CI failure indicated that the minimum version is 16 (Jelly Bean) so this was unneeded. Do we need to keep the check in case someone bumps it down?

Reply:
Oh ok, that makes sense!

> Do we need to keep the check in case someone bumps it down?

Now that you mention it, that would probably be a situation we would want to have covered by it. I'll defer to @blasten though :)